lib/node_modules/@stdlib/blas/ext/base/dnannsum/benchmark/benchmark.js

```diff
@@ -40,19 +42,16 @@ var dnannsum = require( './../lib/dnannsum.js' );
 */
 function createBenchmark( len ) {
 	var out;
-	var x;
-	var i;
+	var x = filledarrayBy( len, 'float64', clbk );
+	out = new Float64Array( 2 );
+	return benchmark;
```

Comment:
Please cross check the conventions here, `x` shall be declared and assigned on separate lines.